spec/operations/save_operation_spec.cr

```diff
@@ -421,6 +421,14 @@ describe "Avram::SaveOperation" do
         end
       end
 
+      it "allows supplying a value that matches the default" do
+        result = Company::SaveOperation.create!(sales: 0_i64, earnings: 0_i64)
```

Comment:
shouldnt this be a float?

```
result = Company::SaveOperation.create!(sales: 0_i64, earnings: 0_f64)
```

Reply:
Oya, I guess you're right. The type extensions have methods for doing the conversation, luckily!

Reply:
TIL.  I am surprised that worked but it makes sense.